Functions.Templates/Bindings/bindings.json

```diff
@@ -209,7 +209,7 @@
         {
           "name": "password",
```

Comment:
rename name as well?

Reply:
Hi @pragnagopa 

Thank you for the review. Probably it is ok for password.   
Let me explain the context. (it is bit confusing) 

The parameter that I want to configure is `password` it is credential for kafka. usually, it used with `username` and `password`  It is used for kafka broker like Confluent cloud. 
Event Hubs has a kafka compatible API. EventHubs only have connection string. If a kakfa client want to connect to EventHubs with Kafka API, it will set `username` as `$ConnectionString` (fixed value)  and `password` as connection string.  So I use `%Password%` as a placefolder. However, it doesn't work on Azure. Probably, reserved word. So that I change the placefolder from `%Password%` to  `%EventHubsConnectionString%` .